scriptworker/constants.py

```diff
@@ -189,8 +189,8 @@
             'project:releng:beetmover:bucket:release': 'all-release-branches',
             'project:releng:googleplay:release': 'release',
             'project:releng:signing:cert:release-signing': 'all-release-branches',
-            'project:releng:googleplay:beta': 'betatest',
-            'project:releng:googleplay:aurora': 'auroratest',
+            'project:releng:googleplay:beta': 'beta',
+            'project:releng:googleplay:aurora': 'nightly',
```

Comment:
Nit: I'd add a comment to explain this discrepancy. This might confuse our future selves.